cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerParcelDecommissionService.java

```diff
@@ -39,59 +39,64 @@
     @Inject
     private ClouderaManagerPollingServiceProvider clouderaManagerPollingServiceProvider;
 
-    public Map<String, String> getParcelsInStatus(ParcelsResourceApi parcelsResourceApi, String stackName, ParcelStatus parcelStatus) {
+    Map<String, String> getParcelsInStatus(ParcelsResourceApi parcelsResourceApi, String stackName, ParcelStatus parcelStatus) {
         try {
-            return getClouderaManagerParcelsByStatus(parcelsResourceApi, stackName, parcelStatus)
-                    .stream()
+            Map<String, String> parcelResponse = getClouderaManagerParcelsByStatus(parcelsResourceApi, stackName, parcelStatus).stream()
                     .collect(toMap(ApiParcel::getProduct, ApiParcel::getVersion));
+            LOGGER.debug("The following parcels are found in {} status: {}", parcelStatus, parcelResponse);
+            return parcelResponse;
         } catch (ApiException e) {
             LOGGER.info("Unable to fetch the list of activated parcels", e);
             throw new ClouderaManagerOperationFailedException("Unable to fetch the list of activated parcels", e);
         }
     }
 
-    public ParcelOperationStatus deactivateUnusedParcels(ParcelsResourceApi parcelsResourceApi, ParcelResourceApi parcelResourceApi,
-            String stackName, Map<String, ClouderaManagerProduct> cmProducts) {
-        Map<String, String> installedComponents = getParcelsInStatus(parcelsResourceApi, stackName, ParcelStatus.ACTIVATED);
-        Map<String, String> filteredParcels = installedComponents.entrySet().stream()
-                .filter(entry -> !cmProducts.containsKey(entry.getKey()))
+    ParcelOperationStatus deactivateUnusedParcels(ParcelsResourceApi parcelsResourceApi, ParcelResourceApi parcelResourceApi,
```

Comment:
I would prefer having public for a method which is called from another class, instead of depending on package-private